docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,120 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
     ```
 
+- **If**
+
+    The if (tensor_if) element allows creating conditional branches based on tensor values. For example, you may skip frames if there is no object detected with high confidence. The input and output stream data type is either `other/tensor` or `other/tensors`.
+
+    - **Properties**
+      - `compared-value`: Specifies the compared value and is represented as operand 1 from input tensors.
+        - A_VALUE: Decided based on a single scalar value.
+	      - TENSOR_AVERAGE_VALUE: Decided based on an average value of a specific tensor.
+	      - CUSTOM: Decided based on a user-defined callback.
```

Comment:
One confusion: 

Is the hierarchy/levels fine ? 
Shouldn't A_Value come in same level as TENSOR_AVERAGE_VALUE
![image](https://user-images.githubusercontent.com/43839737/112898391-670e1080-90fe-11eb-8c5c-5c1b0bd561b6.png)

Reply:
Changed. Thanks